test/unit/test_c_api.cpp

```diff
@@ -5185,6 +5185,108 @@ TEST_F(CApi, datum_ensemble) {
 
 // ---------------------------------------------------------------------------
 
+TEST_F(CApi, check_gtoc_projection) {
```

Comment:
for projections, we rather use the test/gie/ infrastructure. You could add new tests to test/gie/more_builtins.gie for example